### ISSUE TITLE
Uncaught SyntaxError: missing ) after argument list on seemingly okay file

### ISSUE THREAD

**Issue by eankeen** [syumai/dejs#47]
haii,
i receive an 'uncaught syntax error' when templating a seemingly okay file. here is a minimal example

```ts
// test.ts
import * as dejs from 'https://deno.land/x/dejs@0.4.0/mod.ts';

const readerToString = async (reader: Deno.Reader) =>
  new TextDecoder().decode(await Deno.readAll(reader));
const reader = await dejs.renderFile('./test.txt', { baz: 'baz' })
await readerToString(reader)
```

```txt
// test.txt
console.log(`foo bar ${baz}`);
```

```sh
deno run --allow-read test.ts
```

* Error
```txt
error: Uncaught SyntaxError: missing ) after argument list
    at mod.ts:64:12
    at new Promise (<anonymous>)
    at mod.ts:48:11
    at Module.renderFile (mod.ts:168:10)
    at async test.ts:5:16
```

not sure what is going on here

**Comment by syumai** [comment]
This is a bug caused by `eval` used  in dejs implementation.
This must be fixed. Thank you for reporting this.

**Comment by syumai** [comment]
@eankeen https://github.com/syumai/dejs/pull/49/files#diff-e38fb0e81da115ce0eb28e3d2e76d0c1R41
I've added some sanitize rules to render function.
If this looks OK, please close this issue.

**Comment by syumai** [comment]
This feature was released as 0.5.0.
https://github.com/syumai/dejs/releases/tag/0.5.0

**Comment by syumai** [comment]
To support another feature, I've released 0.6.0.
https://github.com/syumai/dejs/releases/tag/0.6.0

**Comment by eankeen** [comment]
@syumai hai, this fixes it for me - thankies!
